crates/harness-tests/src/routing.rs

```diff
@@ -328,28 +329,18 @@ async fn should_prioritize_same_region_ws(ctx: &mut TestRunner) {
 #[tokio::test]
 #[allow(dead_code)]
 async fn test_malformed_header_rsv2_rsv3(ctx: &mut TestRunner) {
-    // What I'm thinking I need to do for this at the most basic level:
-    // 1. Create a new client
-    // 2. Create a WebSocket with a malformed header
-    // 3. Link it to the client
-    // 4. Try to connect to OTT through the balancer
-    // 5. Test for no crash?
-    
-    let mut c = Client::new(ctx).unwrap();
+    let mut client = ClientBuilder::new("ws://localhost:8081/api/room/test").connect_insecure()?; //no method named `connect_insecure` found for enum `Result` in the current scope method not found in `Result<ClientBuilder<'_>, ParseError>`
 
-    let dataframe = DataFrame { // I'm assuming this is a malformed WebSocket
+    let dataframe = DataFrame {
         finished: true,
         reserved: [false, true, true],
         opcode: websocket::dataframe::Opcode::Text,
         data: "{\"action\":\"auth\", \"token\":\"foo\"}"
             .to_string()
             .into_bytes(),
     };
+    
+    client.send_dataframe(&dataframe).await?; // the `?` operator can only be used in an async function that returns `Result` or `Option` (or another type that implements `FromResidual`) the trait `FromResidual<_>` is not implemented for `()`
```

Comment:
same for this comment

Reply:
These were just the errors I was getting when I left off